apps/studio/components/interfaces/DataWarehouse/CreateWarehouseCollection.tsx

```diff
@@ -0,0 +1,98 @@
+import { useParams } from 'common'
+import { useCreateCollection } from 'data/analytics/warehouse-collections-create-mutation'
```

Comment:
can possible re-export it from `data/analytics` so that it isn't such a mouthful